modules/ROOT/pages/reliability-patterns.adoc

```diff
@@ -0,0 +1,164 @@
+= Reliability Patterns
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: reliability, patterns, cluster, ha, transaction, delivery, ensure
+
+A reliability pattern is a design that results in reliable messaging for an application, even if the application receives messages from a non-transactional transport. A reliability pattern couples a reliable acquisition flow with an application logic flow, as shown in the following diagram:
```

Comment:
...even if the application receives messages from a non-transactional _transport_...::connector:: instead?